client/src/Components/SharePost.js

```diff
@@ -23,12 +25,27 @@ const Following = props => {
     fetchFollowData();
   }, [props.user.profileID, props.history]);
 
+  const delPost = post => {
+    authFetch("/posts/" + post._id, null, props, "delete").then(
```

Comment:
You can use authFetch with body parameter set to {} instead of null.
Like the following
`authFetch("/posts/" + post._id, {}, props, "delete"`
I hope this solves your issue.

Reply:
you are right. it worked..